Formula/lbry-sdk.rb

```diff
@@ -0,0 +1,186 @@
+class LbrySdk < Formula
+  include Language::Python::Virtualenv
+
+  desc "Decentralized peer-to-peer protocol for publishing and accessing digital content"
+  homepage "https://lbry.com"
+  url "https://github.com/lbryio/lbry-sdk/archive/v0.91.0.tar.gz"
+  sha256 "6a908089755740fbae8f259ab0397a25f95624b2172fd5a0e005ebde19531c4b"
+  license "MIT"
+  head "https://github.com/lbryio/lbry-sdk.git"
+
+  depends_on "python@3.7"
```

Comment:
Does it not work with a recent python?

Reply:
It does work with a recent Python, but the application currently is specified for python 3.7, that is why the specific dependency on Python 3.7 was used.

Reply:
New Homebrew formula need to work with the latest version of their dependencies. If it doesn't work with 3.9 that means nobody bothered to fix compatibility issues for two years and it would fail: https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff